packages/data-table/stories/DataTable.stories.tsx

```diff
@@ -0,0 +1,137 @@
+import type { StoryObj } from '@storybook/react';
+
+import { Chip } from '@launchpad-ui/chip';
+import { CopyToClipboard } from '@launchpad-ui/clipboard';
+import { Stack } from '@launchpad-ui/stack';
+import { Cell, Column, Row, TableBody, TableHeader } from '@react-stately/table';
```

Comment:
Yup

Reply:
Have you thought about exposing those via the launchpad import? Might be confusing to need to import from the package we consume under the hood.

Reply:
I think it gives a false impression that we maintain those components. That and their naming could clash with our components/exports from the `core` package. You could rename them but ultimately you end up having multiple components that essentially are the same thing. Also similar to `@react-stately/collections`, other packages might expect these as children so having it as a peer dep is more appropriate.

Reply:
Perhaps down the road we can refactor things so that we export them from some supplemental `@launchpad-ui/utils` package that is excluded from core and make that a peer dep?

Reply:
Yeah, or something like `@launchpad-ui/react-stately`? ie. this is a wrapper.